package.json

```diff
@@ -342,18 +345,15 @@
     "pretest": "npm run download:ls && npm run test-compile && npm run lint",
     "test": "node ./out/test/runTest.js",
     "test:unit": "jest",
-    "test:syntax": "npm run test:grammar && npm run test:snap",
```

Comment:
Not directly related to this line, but I'm not able to comment above. Do we want to rename the npm script `download:ls` into something else, since it's now downloading multiple things?

Reply:
Went back and forth on this. Just rename to 'artifacts' or something else? Or do we want separate arguments and then:

```
    "download:artifacts": "ts-node ./build/downloader.ts",
    "download:ls": "ts-node ./build/downloader.ts -thing ls",
    "download:syntax": "ts-node ./build/downloader.ts -thing syntax",
```

Separate files altogether?

Reply:
Assuming we change the reference in `vscode:prepublish` I'd just rename it to `download:artifacts` or anything similar for now, just to avoid confusion - we can address the separation later.

Reply:
`download:artifacts` sounds good to me, too.